### DISCUSSION TITLE
Create macros to allow custom Active Effects with a duration to be added to actors

### DISCUSSION THREAD

**Discussion by Savantford** [Savantford/foundry-weirdwizard#251] [New Feature Requests]
> So not really sure if there is any module that does this sort of thing already, but I'd appreciate it if we could have token markers that don't have any mechanical effects. Ideally ones I can simply add to tokens from the right click menu.  It'd help a great deal with effects like fighter's Combat Controller that allows you to mark enemies. In general I would like to be able to track effects on enemies in an easy to visually represent way. It is nice that the current markers have effects as it makes it fairly easy to apply said effects, but it sadly leaves me with often forgetting other effects because I have no way to visually track them.

**Comment by mhilbrunner** [comment]
I've run into this in the past. What I found best is to use Active Effects, as AEs with a duration also show as token markers. What I used was a macro, where you could enter a name, and optionally select a icon from a list (or enter a URL) and a description, and then a new AE is created with that name/icon/description on the selected tokens.

We could ship such a macro with the system, and even add a button for it to the new Sage tools.

With stuff like https://foundryvtt.com/packages/visual-active-effects, you can even have a nice buff bar to hover and easily read the names and descriptions.
